docs/Pantheon-API/Using-JSON-RPC-API.md

```diff
@@ -71,7 +71,40 @@ Send individual requests as a JSON data package at each prompt:
     `wscat` does not support headers. [Authentication](Authentication.md) requires an authentication token to be passed in the 
     request header. To use authentication with WebSockets, an app that supports headers is required. 
 
-### API Methods Enabled by Default
+## Readiness and Liveness Endpoints 
+
+Pantheon provides readiness and liveness endpoints to confirm the Pantheon node status. Both return a
+`200 OK` status when ready or live and a `503 Service Unavailable` status if not ready or live. 
+ 
+By default, the readiness check requires a peer to be connected and the node to be within 2 blocks of the best
+known block. If [p2p communication is disabled](../Reference/Pantheon-CLI-Syntax.md#p2p-enabled), 
+no peers are required. 
```

Comment:
If there are no remote peers then by definition we have the best chain.  So effectively if p2p communication is disabled, the node is always considered ready.

Reply:
Fixed